qf_lib/tests/unit_tests/trading/emsx/test_emsx_history_service.py

```diff
@@ -0,0 +1,214 @@
+#     Copyright 2016-present CERN – European Organization for Nuclear Research
+#
+#     Licensed under the Apache License, Version 2.0 (the "License");
+#     you may not use this file except in compliance with the License.
+#     You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#     Unless required by applicable law or agreed to in writing, software
+#     distributed under the License is distributed on an "AS IS" BASIS,
+#     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#     See the License for the specific language governing permissions and
+#     limitations under the License.
+
+from datetime import datetime
+from typing import List
+from unittest import TestCase
+from unittest.mock import Mock, patch
+
+import blpapi
```

Comment:
add blpapi as requirement either to pipeline stages, or in requirements.txt

unit tests fail because they cannot find blpapi